From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Mon, 9 Mar 2026 20:53:31 +0100
Subject: [PATCH] Linux 7.0 WIP compat: META

Update the META file to reflect compatibility with the 7.0 kernel to
unblock build of the release candidates.

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 META | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/META b/META
index c6d7e3e94..805ab052f 100644
--- a/META
+++ b/META
@@ -6,5 +6,5 @@ Release:       1
 Release-Tags:  relext
 License:       CDDL
 Author:        OpenZFS
-Linux-Maximum: 6.19
+Linux-Maximum: 7.0
 Linux-Minimum: 4.18
